manifests/role/umich_mailserver.pp

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# The minimum configuration for a host that logically belongs to the
+# University of Michigan. Most--but not all--other roles rely on this
+# one.
+#
+# @example
+#   include nebula::role::umich_mailserver
+class nebula::role::umich_mailserver (
+  $bridge_network = false,
+  $internal_routing = '',
+) {
+
+  class { 'nebula::role::minimum':
+    internal_routing => $internal_routing,
+  }
+
+  if $facts['os']['family'] == 'Debian' and $::lsbdistcodename != 'jessie' {
```

Comment:
This check might not need to be here... I think we'll start seeing these fall away in the coming weeks, and I don't see us applying this role to any jessie hosts.

Reply:
A good point, since this is just for sherry, which is stretch

Reply:
I'll pass on this one until I feel more comfortable with nebula.